WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -160,16 +197,10 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
         return itemsToFulfill
     }
 
-    /// Set the total weight based on the weight of the selected package and
-    /// the products and products variation inside the order items, only if they are not virtual products.
-    ///
-    /// - Parameter initialWeight: An initial value used to set the total package weight if it was input manually.
+    /// Calculate total weight based on the weight of the selected package if it's a custom package;
```

Comment:
nit: could we add here, for the future, that we are using just the custom package because they are the only type with a weight?

Reply:
That makes sense! I'll update the comment to make it clearer.